app/controllers/events_controller.rb

```diff
@@ -3,7 +3,7 @@ class EventsController < ApplicationController
 
   # GET /events
   def index
-    @events = Event.all
+    @events = Event.draft_is false
```

Comment:
Is this a temporary solution? Right now, this is the only place I can access created events from to edit or, in particular, publish them. I guess the "proper" solution would be to toggle this based on user rights.